openproblems/tasks/dimensional_reduction/datasets/__init__.py

```diff
@@ -1 +1,2 @@
-##
+
+from .citeseq import load_citeseq_cbmc
```

Comment:
We should import nesterova here and eventually 10x

Reply:
Also this should say `from .citeseq import citeseq_cbmc`.

Reply:
fixed re: both comments